client/src/styles/layout.css

```diff
@@ -223,3 +225,16 @@ div.jsoneditor {
 ::-webkit-scrollbar-thumb:hover {
   background: rgb(139, 139, 139);
 }
+
+.wrapper-class {
+  padding: 1rem;
+  border: 1px solid rgb(204, 204, 204);
+}
+.editor-class {
+  background-color:rgb(255, 255, 255);
+  padding: 1rem;
+  border: 1px solid rgb(204, 204, 204);
+}
+.toolbar-class {
+  border: 1px solid rgb(204, 204, 204);
+}
```

Comment:
add newline at eof

Reply:
Why is that important to do, is it just a convention? I'll update that now though.